src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -917,6 +917,10 @@
         if edxapp_config.get_bool("email_use_course_id_from_for_bulk")
         else "false"
     ),
+    "bulk-email-default-from-email": (
+        edxapp_config.get("bulk_email_default_from_email")
+        or edxapp_config.require("sender_email_address")
+    ),
```

Comment:
It should use the configured default bulk email address if available.
For QA instances, use the no-reply email address as the default i.e., edxapp:bulk_email_default_from_email: no-reply@edxapp-mail-qa.mitxonline.mit.edu).
Other environments should retain the old configured email address for bulk emails until the new configuration is thoroughly tested and set.